extensions/ql-vscode/src/remote-queries/remote-queries-view.ts

```diff
@@ -129,13 +108,13 @@ export class RemoteQueriesView extends AbstractWebview<
       preserveFocus: true,
       view: "remote-queries",
       additionalOptions: {
-        localResourceRoots: [Uri.file(this.analysesResultsManager.storagePath)],
+        localResourceRoots: [],
       },
     };
   }
 
   protected onPanelDispose(): void {
-    this.currentQueryId = undefined;
+    // No-op
```

Comment:
Why are we keeping this view around instead of removing it?

Reply:
Oh I see you've removed in the next commit. 👍

Reply:
Sorry, I should've specified that I added two commits to this PR.